osu.Game.Modes.Taiko/UI/HitTarget.cs

```diff
@@ -14,6 +14,7 @@ internal class HitTarget : Container
     {
         private const float normal_diameter = TaikoHitObject.CIRCLE_RADIUS * 2 * TaikoPlayfield.PLAYFIELD_SCALE;
         private const float finisher_diameter = normal_diameter * 1.5f;
+        private const float border_offset = 1;

```

Comment:
probably wouldn't hurt to add a comment mentioning which border this is offsetting